test/abstract_unit.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+# Only necessary as long as Ruby 2.6 compatibility is maintained
+require "ruby2_keywords"
```

Comment:
Should this be wrapped in a condition for Ruby < 2.7? It seems to be a no-op in Ruby 3, but I don't know if there is a performance impact to be concerned about:

```
irb(main):001> require "ruby2_keywords"
=> false
```

Reply:
I'm also concerned about making keywords available in tests but not in the whole gem. Is that an okay approach? Should 2.6 support be dropped, or should the require be gem-wide?

Reply:
Why do you need this? I don't see any call to `ruby2_keywords`, and even if there was, it's simpler to do `ruby2_keywords :foo if respond_to?(:ruby2_keywords, true)` than to pull that dependency.

Reply:
Its coming from Drb: https://github.com/rails/activeresource/actions/runs/11599022232/job/32296232186

Reply:
I see, only a problem on 6-0-stable that has been EOL forever, we should drop Rails < 7.